### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Gianni-Nicolas**
**Summary**

Add login.html into the project 

**Screenshoot**

Captura de Pantalla 2020-02-18 a la(s) 16 02 28

**Trello Card**

https://trello.com/c/xaSggaPg/32-login

### CODE REVIEW

File: scss/login.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@import "colors";
+
+%display-config-vertical {
+  align-items: center;
+  display: flex;
+}
+
+.general-login-container{
+	@extend %display-config-vertical;
```
Comment (martinmona) [inline scss/login.scss]: Check the identation

File: scss/login.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	background-color: $light-grey;
+	border-top: 8px solid $blue;
+	@extend %display-config-vertical;
+	flex-direction: column;
+  height: 400px;
+  width: 300px;
+}
+
+.wolox-image-login {
+	max-width: 100%;
```
Comment (martinmona) [inline scss/login.scss]: not necessary

File: scss/login.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.form-label-item {
+  margin-bottom: 10px;
+}
+
+.form-item-login {
+	background-color: $green;
+	border-radius: 5px;
+  color: white;
```
Comment (martinmona) [inline scss/login.scss]: use a variable

File: scss/login.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	height: 30px;
+	margin: 20px 0 10px;
+}
+
+.separator {
+	background: $dark-grey;
```
Comment (martinmona) [inline scss/login.scss]:
```suggestion
	background-color: $dark-grey;
```

**Review by martinmona — CHANGES_REQUESTED** [review]

**Review by martinmona — APPROVED** [review]
chec identation in login.scss

File: scss/login.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	height: 3px;
+	margin-bottom: 10px;
+}
+
+.form-item-signup {
+	border: 1px solid $green;
+	border-radius: 5px;
+	color: $green;
+	height: 30px;
+}
```
Comment (nvescio) [inline scss/login.scss]: Check indentation 😬

**Review by nvescio — CHANGES_REQUESTED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**